ddtrace/ext/ci.py

```diff
@@ -211,15 +216,12 @@ def extract_azure_pipelines(env):
         git.COMMIT_AUTHOR_EMAIL: env.get("BUILD_REQUESTEDFOREMAIL"),
         STAGE_NAME: env.get("SYSTEM_STAGEDISPLAYNAME"),
         JOB_NAME: env.get("SYSTEM_JOBDISPLAYNAME"),
-        # OrderedDict is necessary for comparing against a fixture in testing
         _CI_ENV_VARS: json.dumps(
-            OrderedDict(
```

Comment:
Just curious why is an ordered no longer necessary?

Reply:
Since we are using `json.loads` in order to compare both dictionaries JSON output strings, order does not matter when comparing them.

Relevant code snippet:
``py
b = '{"a": "d", "c": "a", "d": "f"}'
c = '{"c": "a", "a": "d", "d": "f"}'
print(json.loads(b) == json.loads(c)) # True on Python 2.7 and Python 3.10
``